types/webidl-conversions/index.d.ts

```diff
@@ -1,11 +1,10 @@
-// Type definitions for webidl-conversions 6.1
+// Type definitions for webidl-conversions 7.0
 // Project: https://github.com/jsdom/webidl-conversions#readme
 // Definitions by: ExE Boss <https://github.com/ExE-Boss>
+//                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
```

Comment:
This line is no longer necessary, as the minimum supported **TypeScript** version by @DefinitelyTyped is greater than **3.0**.
```suggestion
```

Reply:
done